src/orga/guidelines.rst

```diff
@@ -164,3 +164,21 @@ The guidance that we can provide here is two fold:
    For example, we're holding off on turning off py36 until pypy comes out with pypy3.7.
 3. If there are lots of people in the community relying on older versions, core team can decide to keep an old version around
    For example, we held off turning off py27 even after numpy, scipy dropped support as there were many in the community interested in keeping support until the end of life of that version.
+
+
+Requesting ``core`` to Merge PRs
+================================
+
+Due to various reasons, it is sometimes necessary for the ``core`` team to merge PRs on feedstocks they do not personally
+maintain. The ``core`` team abides by the following guidelines when doing this.
+
+1. The ``core`` team will normally wait three days before merging a PR on a feedstock they do not maintain personally. This
+   rule encompasses both migration PRs from the autotick bot and requests from community members to merge PRs on feedstocks.
+2. In extenuating circumstances (i.e., bad bugs, blocking lots of other PRs/work, unmaintained feedstocks), the ``core`` team reserves the right to merge
+   before the three day window has passed. In this case, at least two ``core`` members must approve the merge. Approvals include
```

Comment:
I think this is confusing two classes of issues. The first are security issues, and the second are maintenance issues. 

The security I can see there being an argument for small number, fiat agreement. The maintenance I think should have the Apache style 3+ & 0-.